tests/test_mounts.py

```
@@ -0,0 +1,138 @@
+               fw.write(somestr)
+
+            with open(path_join(mount_point, "testfile"), "r", encoding="utf8") as fr:
+               self.assertEqual(fr.read(), somestr) 
+
+            mounts._umount(mount_point)
+
+            self.assertFalse(ismount(mount_point))
+
+    def test__mount_bad_dev(self):
+        with self.assertRaises(snapm.SnapmMountError):
+            mounts._mount("/not/a/dev", "/tmp")
```

Ruff (0.14.0)

Probable insecure usage of temporary file or directory: "/tmp"

(S108)